Libplanet/Action/IActionContext.cs

```diff
@@ -98,15 +99,11 @@ public interface IActionContext
         bool BlockAction { get; }
 
         /// <summary>
-        /// Checks whether the specified <paramref name="currency"/> is a native token defined by
-        /// chain's <see cref="Libplanet.Blockchain.Policies.IBlockPolicy{T}.NativeTokens"/>.
+        /// A set of <see cref="Currency"/>s defined as native tokens by chain's
+        /// <see cref="Libplanet.Blockchain.Policies.IBlockPolicy{T}.NativeTokens"/>.
         /// </summary>
-        /// <param name="currency">A token currency to check.</param>
-        /// <returns><see langword="true"/> if the specified <paramref name="currency"/> is a native
-        /// token, otherwise <see langword="false"/>.</returns>
-        /// <seealso cref="Libplanet.Blockchain.Policies.IBlockPolicy{T}.NativeTokens"/>
         [Pure]
-        bool IsNativeToken(Currency currency);
```

Comment:
It would be better to let `IsNativeToken()` left and give it `[Obsolete]` attribute instead.

Reply:
Applying every required changes, safely removed `IsNativeToken()`.